files/stat.py

```diff
@@ -469,15 +519,30 @@ def main():
     # try to get mime data if requested
     if get_mime:
         output['mimetype'] = output['charset'] = 'unknown'
-        filecmd = [module.get_bin_path('file', True), '-i', path]
-        try:
-            rc, out, err = module.run_command(filecmd)
-            if rc == 0:
-                mimetype, charset = out.split(':')[1].split(';')
-                output['mimetype'] = mimetype.strip()
-                output['charset'] = charset.split('=')[1].strip()
-        except:
-            pass
+        mimecmd = [module.get_bin_path('file'), '-i', path]
+        if mimecmd:
+            try:
+                rc, out, err = module.run_command(mimecmd)
+                if rc == 0:
+                    mimetype, charset = out.split(':')[1].split(';')
+                    output['mimetype'] = mimetype.strip()
+                    output['charset'] = charset.split('=')[1].strip()
+            except:
+                pass
+
+    # try to get attr data
+    if get_attr:
+        output['version'] = None
+        output['attributes'] = []
+        attrcmd = [module.get_bin_path('lsattr', False), '-vd', path]
+        if attrcmd:
```

Comment:
This conditional will always be true. The code should be something like:

```python
attrcmd = module.get_bin_path('lsattr')
if attrcmd:
    attrcmd = [attrcmd, '-vd', path]
```

If the command is not found, the module will fail.